examples/tick-ami/tick.json

```diff
@@ -0,0 +1,131 @@
+{
+  "min_packer_version": "0.12.0",
+  "variables": {
+    "aws_region": "us-east-1",
+    "base_ami_name": "tick",
+    "telegraf_version": "1.9.4",
+    "influxdb_version": "1.6.2"
+  },
+  "builders": [{
+    "name": "tick-ami-ubuntu",
+    "ami_name": "{{user `base_ami_name`}}-ubuntu-example-{{uuid | clean_ami_name}}",
+    "ami_description": "An Ubuntu 18.04 AMI that has the TICK stack installed.",
+    "instance_type": "t2.micro",
+    "region": "{{user `aws_region`}}",
+    "type": "amazon-ebs",
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "architecture": "x86_64",
+        "name": "*ubuntu-bionic-18.04-amd64-server-*",
+        "block-device-mapping.volume-type": "gp2",
+        "root-device-type": "ebs"
+      },
+      "owners": ["099720109477"],
+      "most_recent": true
+    },
+    "ssh_username": "ubuntu"
+  },{
+    "name": "tick-ami-amazon-linux",
+    "ami_name": "{{user `base_ami_name`}}-amazon-linux-example-{{uuid | clean_ami_name}}",
+    "ami_description": "An Amazon Linux 2 AMI that has the TICK installed.",
```

Comment:
~~An Amazon Linux 2 AMI that has the TICK installed.~~
An Amazon Linux 2 AMI that has the TICK stack installed.

Reply:
Fixed in #14